BetterUI/Patches/Items.cs

```diff
@@ -83,7 +83,7 @@ public static void UpdateColor(GuiBar durabilityBar, float durability)
 
   static class Stars
   {
-    private static Color starColor = new Color(1.0f, 0.85882f, 0.23137f, 1.0f);
```

Comment:
Oop. VSCode's OmniSharp extension was suggesting some simplifications while I was getting a C# environment setup for the first time and these snuck into this PR.

I can remove these and the `??` null coalescing if you'd like

Reply:
I didn't write the original code and I haven't ever gone through it to clean it up, but these changes are the appropriate way to do it so I'll leave them.